pkg/cmd/admin/node/node.go

```diff
@@ -0,0 +1,117 @@
+package node
+
+import (
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kcmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
+
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+)
+
+const ManageNodeCommandName = "manage-node"
+
+const manageNodeLongDesc = `Manage node operations: schedulable, evacuate and list-pods.
+
+Examples:
```

Comment:
Examples should not be part of the long description, there is an `Example` field in the cmd specifically for them. Use 2 spaces for indentation and `//` for comments. Name the consts `manageNodeLong` and `manageNodeExample`. E.g.:

https://github.com/openshift/origin/blob/master/pkg/cmd/cli/cmd/newapp.go#L32